### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Kira-PH**
Tidying up, replaced Vec3 with Vector3, replaced Mat22 with Matrix3x2

### CODE REVIEW

**Comment by codingben** [comment]
Hey @HughPH 

I see more changes and perhaps more changes will come. What do you think is the best way to review everything? (If we run window-tests/other tests, we see all good)

It’s hard to make sure everything is okay (I mean all changed files).

Do you want me to do 2.0.0-alpha now, and then we'll push more versions?

**Comment by Kira-PH** [comment]
Sure. I just added ChainShape but no collisions. If you cut it now, I think it's at a good place.

A funny thing with nuget is that any version number with - in it is considered pre-release. So it won't appear unless people look for a pre-release version. Then they will be prepared to accept some bugs.

**Comment by Kira-PH** [comment]
Maybe I'll create a more complex test with multiple objects colliding in different ways. Box2D is deterministic, so if we're happy that we think it's right, we can just create a unit test to construct a complex world and test the object locations after n steps.

File: src/box2dx/Box2D.NetStandard/Box2D.NetStandard.csproj [hunk 1]
```diff
@@ -3,15 +3,17 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Authors>Erin Catto</Authors>
-    <Copyright>Copyright © Ihar Kalasouski 2008</Copyright>
-    <PackageTags>Box2D, Physics</PackageTags>
-    <Description>Box2D is the C# port of C++ Box2D. This package supports .NET Standard 2.1.</Description>
+    <Authors>Erin Catto, Ben Ukhanov, Hugh Phoenix-Hulme</Authors>
+    <Copyright>Copyright © Erin Catto, Ben Ukhanov, Hugh Phoenix-Hulme 2020</Copyright>
+    <PackageTags>Box2D, Physics, netcore, netstandard</PackageTags>
```
Comment (codingben) [inline src/box2dx/Box2D.NetStandard/Box2D.NetStandard.csproj]: Can you please change to: <PackageTags>Box2D, Physics, .NET, Standard, Core</PackageTags>

**Comment by codingben** [comment]
> Sure. I just added ChainShape but no collisions. If you cut it now, I think it's at a good place.
> 
> A funny thing with nuget is that any version number with - in it is considered pre-release. So it won't appear unless people look for a pre-release version. Then they will be prepared to accept some bugs.

Haha, yes, I see how nuget works, I think people will see version 2.0.0 alpha.

I see important changes in this PR. Will we do 2.0.0-alpha after you merge this PR or from what we now have in master?

> Maybe I'll create a more complex test with multiple objects colliding in different ways. Box2D is deterministic, so if we're happy that we think it's right, we can just create a unit test to construct a complex world and test the object locations after n steps.

It sounds good. I think we have console tests that look like this.

File: src/box2dx/Box2D.NetStandard/Box2D.NetStandard.csproj (hunk 1 above)
Comment (codingben) [reply src/box2dx/Box2D.NetStandard/Box2D.NetStandard.csproj]: NVM, I changed and will push the changes to master.

**Comment by codingben** [comment]
@HughPH I'm about to make a new package now.

**Comment by Kira-PH** [comment]
OK

File: src/box2dx/Box2D.NetStandard/Box2D.NetStandard.csproj (hunk 1 above)
Comment (codingben) [reply src/box2dx/Box2D.NetStandard/Box2D.NetStandard.csproj]: Pushed.

**Comment by Kira-PH** [comment]
There is a "console test" that shows a ball dropping, but it doesn't prove much. I think we need a scene with every fixture type and every joint type, and as many collisions as possible. I could put something together in RUBE and then we just need a quick visualiser to check it visually which should be easy with DebugDraw. Then if we just run it for n iterations, we can record the positions and use that as our test baseline. If the positions change, we know something is wrong.

**Comment by codingben** [comment]
"error NU5035: The PackageLicenseUrl is being deprecated and cannot be used in conjunction with the PackageLicenseFile or PackageLicenseExpression."

I'll remove this line "PackageLicenseUrl" from the csproj file. Good?

**Comment by Kira-PH** [comment]
Yes fine

**Comment by codingben** [comment]
@HughPH https://www.nuget.org/packages/Box2D.NetStandard/2.0.0-alpha

**Comment by Kira-PH** [comment]
The console test is a bit like that, but it's not automated and it doesn't validate any of the values.

**Pull request merged**